libtock/net/ieee802154.c

```diff
@@ -17,6 +17,11 @@ int libtock_ieee802154_down(void) {
 }
 
 returncode_t libtock_ieee802154_is_up(bool* status) {
+  // For the raw implementation of the 15.4 driver we need to turn on the radio.
+  // If the kernel is managing the 15.4 stack, we likely do not need to do this.
+  // In that case, this will error, but we ignore the error.
+  libtock_ieee802154_command_radio_on();
+
```

Comment:
It seems to me that it may make more sense to not turn the radio on here. This function, as I see it, is just a status getter of the radio. The app can then turn the radio on using the exposed `libtock_ieee802154_radio_on` method if the app desires the radio to be turned on.

Reply:
Yeah I put this in the wrong place. I meant to put it in `libtocksync_ieee802154_up()`.

Reply:
Moved.